### ISSUE TITLE
Summaries can't view in tensorboard

### ISSUE THREAD

**Issue by mrbeann** [tensorforce/tensorforce#537]
I used summaries as in https://tensorforce.readthedocs.io/en/latest/summary_spec.html, the configuratio is like 
```
self.agent = PPOAgent(
            states=states_spec,
            actions=actions_spec,
            network=network_spec,
            summarizer=dict(directory="./board",
                        #steps=50,
                        labels= ["graph", "total-loss", "losses", "rewards", "inputs"]
                    ),
```

after that, I can see the graph in tensorboard, but can't get the loss and other values, Is there anything wrong?

**Comment by AlexKuhnle** [comment]
Are you on the most recent Github version? I know that some labels are currently not working, for not entirely clear TensorFlow-related reasons, but at least `total-loss` should work, as far as I remember.

**Comment by mrbeann** [comment]
Yes, I used the newest version. But I can't see any numbers.

**Comment by AlexKuhnle** [comment]
When using the example script and configs as follows, and adding `"directory": "tensorboard"` for `"summarizer"` in `vpg.json`, I can see both the graph (under Graphs) and a plot of `total-loss` (under Scalars) in TensorBoard (via `tensorboard --logdir tensorboard`). However, I expect that the labels `"rewards"` and `"inputs"` do not currently work.

```
python examples/openai_gym.py CartPole-v1 -a examples/configs/vpg.json -n examples/configs/mlp2_network.json -e 2000 -m 200
```

**Comment by mrbeann** [comment]
Quite strange, I just use 
```
summary = dict(directory="./board",
                        #steps=50,
                        labels= ["graph", "total-loss", "losses"]
                    )
```
but it still not working.

**Comment by AlexKuhnle** [comment]
Actually, it may be worth trying the directory value with `./` removed, at least I seem to remember that someone else had a problem with that at some point. Otherwise I'm not too sure what could be wrong, but there will be a reworked version with hopefully improved summaries-handling coming soon, so that may change things.

**Comment by mrbeann** [comment]
Thanks for your answer, but it still doesn't work. Is there another way I can check the loss to see if it converge?

**Comment by AlexKuhnle** [comment]
A hacky way would be to add a print statement [here](https://github.com/tensorforce/tensorforce/blob/master/tensorforce/models/memory_model.py#L425):  `loss = tf.Print(loss, [loss])`, but obviously that's far from ideal. Ideally the new version fixes the problem for you.

**Comment by AlexKuhnle** [comment]
Summaries work with the new branch (`major-revision`), which will be merged with master soon, hence already closing summary-related issues. Currently supported labels:
```python
"bernoulli", "beta", "categorical", "distributions", "dropout", "entropy", "gaussian", "graph",
"loss", "losses", "objective-loss", "regularization-loss", "relu", "updates", "variables"
```
Others will be added, and please feel free to post specific requests here.
